source/includes/usage-examples/write-code-examples.java

```diff
@@ -0,0 +1,77 @@
+# start-insert-one
+Document document = new Document("<field name>", "<value>");
+Mono<InsertOneResult> result = Mono.from(collection.insertOne(document));
+
+System.out.printf("Inserted 1 document with ID %s%n.", 
+      result.block().getInsertedId());
+# end-insert-one
+
+# start-insert-multiple
+Document doc1 = new Document("<field name>", "<value>");
+Document doc2 = new Document("<field name>", "<value>");
+
+List<Document> documents = Arrays.asList(doc1, doc2);
+
+Mono<InsertManyResult> result = Mono.from(collection.insertMany(documents));
+
+System.out.printf("Inserted documents with IDs %s%n.",
+      result.block().getInsertedIds());
+# end-insert-multiple
+
+# start-update-one
+Mono<UpdateResult> result = Mono.from(collection.updateOne(
+      eq("<field name>", "<value>"),
+      set("<field name>", "<new value>")));
+
+System.out.printf("Inserted %s document with ID %s%n.",
+      result.block().getModifiedCount(), result.block().getUpsertedId());
+# end-update-one
+
+# start-update-multiple
+Mono<UpdateResult> result = Mono.from(collection.updateMany(
+      eq("<field name>", "<value>"),
+      set("<field name>", "<new value>")));
+
+System.out.printf("Inserted %s documents with ID %s%n.",
+      result.block().getModifiedCount(), result.block().getUpsertedId());
```

Comment:
This is not a right message for `updateMany`.

Reply:
1. `getUpsertedId` returns `null` because the code did not upsert anything. You can use `getModifiedCount` instead.
2. I propose re-runnign all examples and making sure that the printed messages make sense.